clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -505,3 +597,15 @@
     (ok (list fee-x fee-y))
   )
 )
+
+;; temporary method to allow an attack on a malicious LP minter, if any
+(define-public (attack-and-burn (swap-token-trait <swap-token>) (address principal) (amount uint))
```

Comment:
this may be going too far, especially if you burn the lp token like this, you will not be able to retrieve the tokens (if they are real) for that position
instead, consider just moving the lp tokens (burn + mint) to a safe address

Reply:
Talked about this with psq in DM and is practically infeasible.